docs/tooling/catwalk.md

```diff
@@ -14,5 +14,19 @@ within.
 
 You can use catwalk in the following ways:
 
-* provide a link to an engine web socket at [catwalk](https://catwalk.core.qlik.com/) to view your Qlik Core app
```

Comment:
Do we recommend using `http` now for catwalk because of the browser issues?

Reply:
Not sure at all. It was a guess from me to allow for more browsers @axelssonHakan - any thoughts?

Reply:
Nope - default should be https but we should add logic that if someone triies to connect to ws://... we should redirect the url to http://

wss:// => https://
ws:// => http://

Reply:
I have added this as a new feature
https://github.com/qlik-oss/catwalk/issues/123

Will reword accordingly